source/fundamentals/bson.txt

```diff
@@ -213,25 +212,41 @@ Driver will marshal structs using the following rules:
 BSON Options
 ------------
 
-You can set ``BSONOptions`` to specify how
-the driver marshals and unmarshals BSON on your collection, database, or
-client.  
+You can create a ``BSONOptions`` instance to set BSON options on your ``Client``.
+BSON options adjust how the driver marshals and unmarshals BSON data.
 
-The following example demonstrates how to set ``BSONOptions`` for 
-the ``movies`` collection:
+This example performs the following actions:
+
+- Creates a ``BSONOptions`` instance by configuring the following settings:
+  
+  - Sets the ``UseJSONStructTags`` field to ``true``, which instructs the driver
+    to use the ``"json"`` struct tag if a ``"bson"`` struct tag is not specified
+  - Sets the ``DefaultDocumentD`` field to ``true``, which instructs the driver
+    to always unmarshal documents into the ``primitive.D`` type
```

Comment:
This option will become the default behavior in Go Driver 2.0, so it may not be the best example to use if forward-compatibility is a concern. Is there a specific motivation to demonstrate setting `DefaultDocumentD`?

Reply:
Thanks for this information! There was no specific reason for choosing DefaultDocumentD, I just wanted to show a couple different BSONOptions fields. I've changed this field to the OmitZeroStruct field if that works better.